cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/lib/kv/LevelDBKVTableTest.java

```diff
@@ -38,15 +39,20 @@
   @ClassRule
   public static TemporaryFolder tmpFolder = new TemporaryFolder();
 
-  static LevelDBTableService service;
+  private static LevelDBTableService levelDBTableService;
 
   @BeforeClass
   public static void init() throws Exception {
     CConfiguration conf = CConfiguration.create();
     conf.set(Constants.CFG_LOCAL_DATA_DIR, tmpFolder.newFolder().getAbsolutePath());
     conf.set(Constants.CFG_DATA_LEVELDB_DIR, tmpFolder.newFolder().getAbsolutePath());
-    service = new LevelDBTableService();
-    service.setConfiguration(conf);
+    levelDBTableService = new LevelDBTableService();
+    levelDBTableService.setConfiguration(conf);
+  }
+
+  @After
+  public void clear() {
+    levelDBTableService.clearTables();
```

Comment:
don't you rather want to call close()?

Reply:
Since the service is like a singleton object, I don't want to call close, which makes the service unusable after closing it.
`clearTables` is very similar.